app/controllers/api/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::ReviewsController < ApplicationController
+  respond_to :json
+  skip_before_action :authenticate
+
+  def create
+    review = Review.new(review_params)
+
+    if review.save
+      respond_with review
+    else
+      head :unprocessable_entity
+    end
+  end
+
+  def show
+    review = Review.find(params[:id])
+    respond_with review
```

Comment:
Yea I tried it with specifying the namespace as well.  I'll look back into it.  If you don't see what it is right off maybe there is something more substantial going on.